ivy/functional/frontends/jax/numpy/logic.py

```diff
@@ -153,6 +153,11 @@ def isfinite(x, /):
     return ivy.isfinite(x)
 
 
+@to_ivy_arrays_and_back
+def isin(element, test_elements, assume_unique=False, invert=False):
+    return ivy.isin(element, test_elements, assume_unique=assume_unique, invert=invert)
```

Comment:
Hello, so this Ivy function is currently in the experimental API  which means that it can't be called until it's been revised and moved into the functional API.

In order for this to be able to merge, you will need to implement ivy.isin() here with a composition of current functional API functions.

Hint: This function has already been implemented as a composition of Ivy functions 

Thanks,
James